totalRP3/core/impl/slash.lua

```diff
@@ -131,7 +131,8 @@ local function rollDice(diceString)
 
 		total = total + modifierValue;
 
-		Utils.message.displayMessage(loc.DICE_ROLL:format(Utils.str.icon(TRP3_API.globals.is_classic and "inv_enchant_shardglowingsmall" or "inv_misc_dice_02", 20), num, diceCount, total));
+		local modifierString = (modifierValue ~= 0) and ((modifierValue > 0 and '+')..(modifierValue)) or ''; -- we add a + to positive modifiers and don't render a 0 value
```

Comment:
The following would be shorter and equivalent:
```lua
local modifierString = (modifierValue == 0) and "" or format("%+d", modifierValue);
```

![image](https://user-images.githubusercontent.com/287102/96701096-1f819c80-1388-11eb-97b3-8091000ee993.png)